api/v1beta1/openstackdeploy_types.go

```diff
@@ -54,16 +54,43 @@ const (
 	DeployCondReasonConfigCreate ConditionReason = "ConfigCreate"
 )
 
+// OpenStackAnsibleSettingsSpec defines advanced ansible-playbook settings
+type OpenStackAnsibleSettingsSpec struct {
+	// +kubebuilder:validation:Optional
+	// Playbook to run from config-download
```

Comment:
Should we also specify that it runs the deployment playbook be default if it is unset?

Reply:
Also, should we validate with an Enum for all the valid options you have in the case statement below?

Reply:
> Should we also specify that it runs the deployment playbook be default if it is unset?

Sure, can explain how this works with the mode param

> Also, should we validate with an Enum for all the valid options you have in the case statement below?

I figured it would be better not to restrict the advanced ("I know what I'm doing") playbook option as I've not covered some of the other cases yet:
```
$ ls tripleo-ansible/*playbook*
tripleo-ansible/common_deploy_steps_playbooks.yaml   tripleo-ansible/external_upgrade_steps_playbook.yaml     tripleo-ansible/scale_playbook.yaml
tripleo-ansible/deploy_steps_playbook.yaml           tripleo-ansible/post_upgrade_steps_playbook.yaml         tripleo-ansible/update_steps_playbook.yaml
tripleo-ansible/external_update_steps_playbook.yaml  tripleo-ansible/pre_upgrade_rolling_steps_playbook.yaml  tripleo-ansible/upgrade_steps_playbook.yaml
```